packages/ui/src/components/counter-badge.tsx

```diff
@@ -0,0 +1,42 @@
+import { cn } from '@utils/cn'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+const counterBadgeVariants = cva('badge badge-counter inline-flex w-fit items-center', {
+  variants: {
+    theme: {
+      default: 'badge-muted',
+      info: 'badge-info'
+    }
+  },
+  defaultVariants: {
+    theme: 'default'
+  }
+})
+
+type CounterBadgeProps = Omit<
+  React.HTMLAttributes<HTMLDivElement>,
+  'color' | 'role' | 'aria-readonly' | 'tabIndex' | 'onClick'
+> & {
+  theme?: VariantProps<typeof counterBadgeVariants>['theme']
+}
+
+function CounterBadge({ className, theme = 'default', children, ...props }: CounterBadgeProps) {
+  return (
+    <div
+      aria-readonly="true"
+      tabIndex={-1}
```

Comment:
Since this is not an interactive element, why do we need to try to prevent it being focused (it won't be anyway)?

Reply:
removed